draft-ietf-taps-interface.md

```diff
@@ -2380,11 +2383,27 @@ Default:
 
 This property specifies that a message should be sent and received as a single
 packet without network-layer fragmentation, if possible.
-Attempts to send a message with this property set to a size greater than the
+This only takes effect when used with transports supporting this functionality
+and running over IP version 4. When it does take effect, setting this property to
```

Comment:
Why only IPv4? IPv4 and IPv6 stacks can perform host fragmentation.
I'd expected: "When it does take effect, setting this property to true will inhibit network layer fragmentation by the endpoint, and will cause the IPv4 to set the Don't Fragment bis in the IP header, ...

Reply:
I think the concern was about making a false promise for the IPv6 case. As long as we say that the system will attempt to do this, that's okay.

Reply:
I'm following your suggestion above, which eliminates the IPv4 part of the sentence too, so I think this is settled.